### ISSUE TITLE
Add basic python programs in python 2.x

### ISSUE THREAD

**Issue by keshav2904** [keshav2904/myPythonCodes#1]
any basic python 2 program like Fibonacci series, adding two numbers etc.

**Comment by lilcoltsmith** [comment]
@keshav2904 , if you are okay with people participating in the Hacktoberfest event contributing, could you please add the " hacktoberfest" topic to this repository?

**Comment by keshav2904** [comment]
Yeah sure. But how do i add hacktoberfest topic to my repository?

**Comment by lilcoltsmith** [comment]
@keshav2904 , here is a quick video tutorial. This will also help others find your repo so they can contribute.

https://youtu.be/FxBv4FMcDgo

**Comment by 0xYashwanth** [comment]
Hey @keshav2904, I have a pretty decent knowledge of python. I want to contribute to  Basic python programs. Could you please assign this to me?
Thanks for reading.
